src/cljs/athens/electron.cljs

```diff
@@ -325,6 +325,52 @@
   ;; open or create a new starter db
 
   ;; Watch filesystem, e.g. in case db is updated via Dropbox sync
+  
+    ;; Update index.transit    
+  (defn write-db-index [file filepath]
+    (go   
+     (when-not (exists? (fs.writeFileSync filepath file))  
+     (js/alert "Backup file correctly restored, click Ok to re-start") 
+     (js/setTimeout #(dispatch [:boot/desktop]) 50))    
+    ) 
+  )
+  
+  ;; Create new index.transit db
+  (defn create-db-index [file filepath]
+    (fs.writeFile filepath file (fn [e] (dispatch [:fs/open-dialog])))
+  ) 
+  
+  (defn open-dialog-index
+    "Allow user to open a Backup file."
+    [filepath]
+    (js/alert "Your index.transit is corrupted, please open up a backup")
+    (let [res  (.showOpenDialogSync dialog (clj->js {:properties ["openFile"]
+                                                     :filters    [{:name "Transit" :extensions ["transit.bkp"]}]}))
+          open-file (first res)]
+      (when (and open-file (.existsSync fs open-file))
+        (let [read-db (.readFileSync fs open-file)
+              db-file      (try  (dt/read-transit-str read-db)(catch  :default e (do (js/console.error (js/Error. e)) 
+                                                                                     (open-dialog-index filepath))))              
+              ] 
+           (if (= (:schema db-file) db/schema) 
+                  (write-db-index read-db filepath)(open-dialog-index filepath))))
```

Comment:
Misleading formatting

Reply:
@sunydev it appears that you can execute the `./script/style` file (if you're on a UNIXy platform; have `bash`) to generate and view a styling report which'd likely make the violations clearer, so more direct to address.

Reply:
yep @phrohdoh  thanks i ran the script and some lines are in red but for some reasons the answer is exactly the same as before ??